src/pyorcidator/helper.py

```diff
@@ -18,15 +19,25 @@
     "Loop profile": "P2798",
     "Scopus Author ID": "P1153",
     "ResearcherID": "P1053",
+    "github": "P2037",
 }
 
 
-def get_external_ids(data):
-    id_list = data["person"]["external-identifiers"]["external-identifier"]
-    id_dict = {}
-    for id in id_list:
-        id_dict[id["external-id-type"]] = id["external-id-value"]
-    return id_dict
+def get_external_ids(data) -> Mapping[str, str]:
+    """Get external identifiers that can be mapped to Wikidata properties."""
+    rv = {}
+    for d in data["person"]["external-identifiers"]["external-identifier"]:
+        rv[d["external-id-type"]] = d["external-id-value"]
+    for d in data["person"]["researcher-urls"].get("researcher-url", []):
+        url_name = d["url-name"].lower().replace(" ", "")
+        url = d["url"]["value"].rstrip("/")
+        if url_name == "github" and "github.com" in url:
+            github = url[len("https://github.com/")]
```

Comment:
I've noticed there's a small bug here. You probably want `github = url[len("https://github.com/"):]`. Notice the colon

Reply:
Indeed, I think I fixed this in c9e7f9c75180cd50350e099d613d778c9f74a82f

Reply:
Yep, thanks!